tests/_helpers.py

```diff
@@ -2,7 +2,8 @@
 
 import pytest
 
-def import_(library):
-    if 'cupy' in library:
+
+def import_or_skip_cupy(library):
```

Comment:
FYI, I'm probably going to rename this back, because I need to add some additional skipping logic for jax as well at #84

Reply:
Sure. I think it would  be good to keep the "or skip" in the name, because from looking at the places where it was used, it was not clear what it does.